src/http/routing.rs

```diff
@@ -345,6 +351,10 @@ impl Route {
         format!(api!("/channels/{}/messages/{}"), channel_id, message_id)
     }
 
+    pub fn channel_message_crosspost(channel_id: u64, message_id: u64) -> String {
```

Comment:
Please document this function.

Reply:
Others functions are undocumented, I don't think this one should be documented